libsignal/src/main/java/org/session/libsignal/crypto/MnemonicCodec.kt

```diff
@@ -24,7 +24,7 @@ class MnemonicCodec(private val loadFileContents: (String) -> String) {
             internal val truncatedWordSetCache = mutableMapOf<Language, List<String>>()
         }
 
-        internal fun loadWordSet(): List<String> {
+        internal suspend fun loadWordSet(loadFileContents: suspend (String) -> String): List<String> {
```

Comment:
Perhaps to keep `Language` (or `Configuration` if we delete Language or merge these 2) a simple data class `loadWordSet` and `loadTruncatedWordSet` would appear to better live in the `MnemonicCodec` where we will already have access to `loadFileContents`